tests/testFileMap_existence.py

```diff
@@ -0,0 +1,133 @@
+"""
```

Comment:
I thought about this a little more. The majority (all without the commented block) of the asserts here are actually testing the functionality of `sims_utils` not `sims_catUtils`'   I know we need to make sure that users have the new sed library installed before they go accessing the LSSTCATSIM stars.  But unit tests aren't REALLY the best way to check version dependencies. It seems to me like the cleanest way would be to add a version requirement to the .table file like `setupRequired(sims_sed_library >= 1.2.3)`  (though like you were saying currently the only explicit dependency on sims_sed_library is in photUtils, so you'd then have either require that catUtils depends on the version of photUtils that depends on the new sims_sed_library OR make catUtil's dependency on sims_sed_library explicit.  I vote for making the sims_catUtil's dependency on sims_sed_library explicit, since that was intention behind this unit test.

The other day you might have been telling me that we can't version sims_sed_library until we move it to lfs...  If that is the case, perhaps we should add a note here that this unit test is just a temporary way to check the version of the sed_library.